uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -221,4 +223,12 @@ protected void initialiseAppImpl() {}
     public void handleFirstRun() {
         UiAutomatorUtils.handleButtonClick("com.msft.identity.client.sample.local:id/btnStartTask");
     }
+
+    // Handles first run of the app based on the user account type to be used.
+    public void handleFirstRunBasedOnUserType(UserType userType) throws UiObjectNotFoundException {
+        handleFirstRun();
+        if (userType == UserType.MSA) {
+            selectFromConfigFile("MSA");
+        }
+    }
```

Comment:
Is this method needed? Seems more clear to use `handleFirstRun()` then ` selectFromConfigFile("MSA")` in the test case itself, that way devs can see exactly what is happening.

Reply:
IMO, selecting a config based on account type should be part of handling first run experience rather than explicitly selecting it in every test case.

Reply:
Hmmm, I think that's fair. I also realize now that you are moving towards parameterized tests, so this approach will help keep tests uncluttered. Let's go with your existing implementation.